labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -59,7 +59,7 @@ object ListManipulationExercise01 {
    *  - ... etc 
    */
   def concatLists[T](l1: List[T], l2: List[T]): List[T] = {
-    error("fix me")
+    l1.union(l2)
   }
 
```

Comment:
According to the documentation, for lists at least it puts everything into the union in the order they originally appeared. This passed the test case, anyway. It's probably sort of an interface method, so it might do something totally different for other set-like things but for lists it works like concatenation.